src/UI/Movies/MoviesCollectionClient.js

```diff
@@ -0,0 +1,262 @@
+var _ = require('underscore');
```

Comment:
nevermind.. this file is not needed andwillbe removed in a subsequent commit.... Im finally getting a handle on all this..